config/default.yml

```
@@ -2669,6 +2669,7 @@ Lint/UselessNumericOperation:
 Lint/UselessOr:
   Description: 'Checks for useless OR expressions.'
   Enabled: pending
+  SafeAutoCorrect: false
   VersionAdded: '1.76'
```

It would be better to take precautions so that unfinished code is not unexpectedly deleted by the LSP. It also seems advisable to add `VersionChanged` since this alters the default behavior.

```suggestion
  SafeAutoCorrect: false
  AutoCorrect: contextual
  VersionAdded: '1.76'
  VersionChanged: '<<next>>'
```

---

I just learned about `AutoCorrect: contextual` for the first time, thank you 👍 
You’re absolutely right. I’ll update it accordingly.